src/timestamp.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+
+  // Created a function to get the date
+  function toTimestamp(strDate) {
+    const now = new Date();
+    const date = [now.getMonth() + 1, now.getDate(), now.getFullYear()];
```

Comment:
Reduce complexity of building the timestamp with .getTime()

http://stackoverflow.com/questions/221294/how-do-you-get-a-timestamp-in-javascript